### PR TITLE
feat(svelte): adding svelte cli generator

### PR DESCRIPTION
**Pull request by Armanio**
Issue: https://github.com/storybooks/storybook/issues/6071

Disclaimer: i'm don't write on svelte some time, i need review 'button' component. Maybe issue author can look? @equinusocio? 
Result svelte storybook its very simple: has only button component. We need 'welcome' component?

## What I did
I just added svelte generator.
## How to test
Run `npx -p @storybook/cli sb init --type svelte`

- Is this testable with Jest or Chromatic screenshots?
I didn't write test. Maybe someone help me? 
- Does this need a new example in the kitchen sink apps?
No.
- Does this need an update to the documentation?
Yes. Here in sidebar need add link to '/svelte/guide'

### CODE REVIEW

**Comment by equinusocio** [comment]
@Armanio PS:There already is the link to the svelte guide (https://storybook.js.org/docs/guides/guide-svelte/) 👍

**Comment by equinusocio** [comment]
@Armanio Does this will support the `template` key for svelte components?

**Comment by Armanio** [comment]
As i know, storybook support svelte component. 
This pr just add cli generator. Nothing more. :man_shrugging:

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
Looking amazing!

**Review by ndelangen — APPROVED** [review]

File: lib/cli/package.json [hunk 1]
```diff
@@ -44,32 +44,32 @@
...
+    "@storybook/ember": "5.1.0-alpha.9",
+    "@storybook/html": "5.1.0-alpha.9",
+    "@storybook/marko": "5.1.0-alpha.9",
+    "@storybook/mithril": "5.1.0-alpha.9",
+    "@storybook/polymer": "5.1.0-alpha.9",
+    "@storybook/preact": "5.1.0-alpha.9",
+    "@storybook/react": "5.1.0-alpha.9",
+    "@storybook/react-native": "5.1.0-alpha.9",
+    "@storybook/riot": "5.1.0-alpha.9",
     "@storybook/svelte": "5.1.0-alpha.8",
```
Comment (shilman) [inline lib/cli/package.json]: @Armanio also needs update?

**Review by shilman — CHANGES_REQUESTED** [review]

File: lib/cli/package.json (hunk 1 above)
Comment (Armanio) [reply lib/cli/package.json]: Of course. Fixed!

**Review by shilman — APPROVED** [review]

File: lib/cli/generators/SVELTE/template/.storybook/config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { configure } from '@storybook/svelte';
+
+// automatically import all files ending in *.stories.js
+const req = require.context('../stories', true, /.stories.js$/);
```
Comment (shilman) [inline lib/cli/generators/SVELTE/template/.storybook/config.js]: @Armanio I think there was a PR recently to turn `/.stories.js$/` into `/\.stories\.js$/` globally. Can you look for that and make the update here if it makes sense?

**Review by shilman — CHANGES_REQUESTED** [review]
see inline comment about regex

File: lib/cli/generators/SVELTE/template/.storybook/config.js (hunk 2 above)
Comment (Armanio) [reply lib/cli/generators/SVELTE/template/.storybook/config.js]: Done.

**Review by shilman — APPROVED** [review]

**Pull request merged**